packages/data-point/src/reducer-helpers/map/map.test.js

```diff
@@ -0,0 +1,38 @@
+const { ReducerMap } = require("./map");
+const { Accumulator } = require("../../Accumulator");
+const { resolve } = require("../../resolve");
+
+describe("ReducerMap", () => {
+  const spec = value => value * 2;
+
+  describe("constructor", () => {
+    it("should set type to 'constant'", () => {
+      const reducer = new ReducerMap(spec);
+      expect(reducer.type).toEqual("map");
+    });
+
+    it("should create a reducer from spec and assign to mapReducer", () => {
+      const reducer = new ReducerMap(spec);
+      expect(reducer.mapReducer).toHaveProperty("type", "function");
```

Comment:
suggestion to rename `mapReducer` to simplify `reducer`. I think the current name is misleading, because it sounds like it actually calls Array.map, but it doesn't